.changeset/great-news-beg.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-react": patch
+---
+
+Adds `inline-size` to `Tabs.Wrapper` to counteract visual change to the width of horizontal tabs wrappers.
```

Comment:
Changesets should be worded in a way that emphasizes why the user would care about this change.

```suggestion
Fixed a regression in `Tabs` where the panel content was not occupying the full width of the container.
```

Reply:
Changed along with the css changeset in this [commit.](https://github.com/iTwin/iTwinUI/pull/1955/commits/8177c00df7a68b47737f9f2b92017bb2ecb84aee)